vitest.config.ts

```diff
@@ -20,10 +21,64 @@ export default defineConfig({
     __VUE_PROD_DEVTOOLS__: 'false',
     __VUE_PROD_HYDRATION_MISMATCH_DETAILS__: 'true',
   },
+  cacheDir: resolve(dirname(fileURLToPath(import.meta.url)), 'node_modules/.vite'),
   test: {
     reporters: 'dot',
     coverage: {
       exclude: ['./packages/**/demo.vue'],
     },
+    clearMocks: true,
+    workspace: [
+      'packages/*/vitest.config.ts',
+      {
+        // add "extends: true" to inherit the options from the root config
+        extends: './vitest.config.ts',
+        resolve: {
+          alias: {
+            vue: 'vue/dist/vue.esm-bundler.js',
+          },
+        },
+        test: {
+          include: ['packages/**/*.browser.{test,spec}.ts'],
+          name: 'browser',
+          setupFiles: ['vitest-browser-vue'],
+          browser: {
+            enabled: true,
+            provider: 'playwright',
+            headless: true,
+            instances: [
+              { browser: 'chromium' },
+              { browser: 'firefox' },
+              { browser: 'webkit' },
+            ],
+          },
+        },
+      },
+      {
+        extends: './vitest.config.ts',
+        test: {
+          name: 'unit',
+          environment: 'jsdom',
+          setupFiles: [resolve(__dirname, 'packages/.test/setup.ts')],
```

Comment:
```suggestion
          setupFiles: [resolve(import.meta.dirname, 'packages/.test/setup.ts')],
```

Change them as well in the packages paths above